Project1/Project1/NetworkController.swift

```diff
@@ -0,0 +1,30 @@
+////
+////  NetworkController.swift
+////  Project1
+////
+////  Created by Golos on 07.05.2021.
+////
+//
+import Foundation
+
+class NetworkController {
+    
+    static func getRequest(searchItem: String, completion: @escaping (Response) -> Void) {
+        guard let url = URL(string: "https://content.guardianapis.com/search?q=\(searchItem)&api-key=add4e59e-5c6d-460f-b8e9-458cf0e68b86&pageSize=20") else { return }
+        let session = URLSession.shared
+        session.dataTask(with: url) { (data, response, error) in
+            guard let data = data else { return }
+            do {
+                let json = try JSONSerialization.jsonObject(with: data, options: [])
```

Comment:
т.к. ошибку ты все равно особо не обрабатываешь, предлагаю заменить `do/catch` на просто еще одну строку в guard c `try?`, по типу (оступы поправь, у меня редактор поплыл немного):
```suggestion
            guard let data = data, 
                    let json = try? JSONSerialization.jsonObject(with: data, options: []) else { return }
```